style.css

```diff
@@ -133,343 +110,146 @@ section {
   list-style-type: none;
 }
 
-/* secttion two area */
-.section-two {
-  background-image: none;
-  width: 327px;
+/* The section area */
+
+.case1 {
+  display: flex;
+  justify-content: center;
+  padding: 50px 20px;
+  background-color: #e5e5e5;
 }
 
-.section-two .background {
-  border-radius: 10px;
-  box-shadow: 3px 3px 8px #888;
-  margin-left: 20px;
-  margin-right: 20px;
-  height: 500px;
-  padding-left: 10px;
-  padding-top: 15px;
-  background: #fff;
+.case1 .case2 {
+  padding: 20px;
+  background-color: #fff;
+  border-radius: 20px;
 }
 
-.section-two .background div img {
-  width: 263px;
-  height: 220px;
+.case1 .case2 ul {
+  display: flex;
```

Comment:
- [ ] Kindly make sure to add the space between the description content in order to meet the provided Figma design :wink: 
- [ ] KIndly make sure to use the same color text the same as the provided design.
  
     - ### Your design
     ![image](https://user-images.githubusercontent.com/87197412/199298573-9eeca3d4-da21-414d-a3a1-23ebc48ed949.png)


    - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/199298788-969be5de-d1f2-4113-874d-81aea3b87f17.png)